docs/arch/adr-24.md

```diff
@@ -0,0 +1,64 @@
+# ADR 24: Drawing Shape Mark Annotation Models
+
+Created: December 18, 2019
+
+## Context
+
+Panoptes-Front-End's drawing tools largely developed from previous custom projects and/or were added one at a time to support a specific project. Because of this, several inconsistencies have been discovered in downsteam analysis and aggregation. To explain the inconsistencies, a few definitions are needed:
+
+- RHC: A right handed coordinate system, this is defined as a system where positive angles rotate an object from the +x axis to the +y axis with angle=0 along the +x axis
+
+- LHC: A left handed coordinate system, this is defined as a system where positive angles rotate an object from the +x axis to the -y axis with angle=0 along the +x axis
+
+- Domain: The range of values a number can take [ or ] is inclusive, ( or ) is exclusive.
+
+- Upper origin: The point 0, 0 is in the upper left of the plot
+
+- Lower origin: The point 0, 0 is in the lower left of the plot
+
+The inconsistencies comprise of:
+
+- The browser SVG coordinate systems use _RHC_ with an _upper origin_ resulting in positive angles rotating clockwise. Most plotting software (R, Python, Matlab) are _RHC_ with a _lower origin_ resulting in positive angles rotating counter-clockwise. 
+  - The position of origin has been inconsistent between tools which has an effect on the final annotation too. Most use the center x, y point, but some don't
+  - Some of the drawing tools use _LHC_
+- Some tools' annotation use `angle` some use `rotation`
+- It's unclear when the x, y annotation refers to the center point of the shape
+- It's unclear when the x, y annotation is being used as the point of rotation
+
+Some of the mark annotation models have a few other issues as well:
+
+- Some shapes have default values which an create bias. For example, the ellipse has a default axis ratio of 0.5 and many volunteers have left the default creating a bias ([comment](https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-516788821))
+- The freehand drawing tool has peformance impact on the browser as the drawing is being created and with the job to create classification exports as well. This is because the current annotation consists of every single x, y point created
+
+## Decision
+
+The shape's mark annotation models should change for consistency and improved post-classification analysis in the following ways:
+
+- The annotation should use the mathematical standard of _RHC_ with a domain of `(-180, 180]` for consistent angle calculation
```

Comment:
```suggestion
- The annotation should use the mathematical standard of _RHC_ with a domain of `[-180, 180]` for consistent angle calculation
```

Reply:
@CKrawczyk you recommended `(-180, 180]` in your original comment in #500, but in a later comment suggested `[180, -180]` (see: https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-563251422). Could you confirm if `-180` should be inclusive or exclusive?

Reply:
Sorry for the confusion, taking a look at the `atan2` implementation in JS (same as python and C) it uses `[-180, 180]`, where the only way to get `-180` is for the "delta x" to be `-inf`.  See the examples section of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan2 for the edge cases around -180 and 180.

Reply:
Thanks for the confirmation.

Reply:
Looking at JS implementation of `Math.atan2` (same as that used in Python and C) the range is `[-180, 180]`, with the value of `-180` only being used with "delta x" is `-inf` (something that can't happen with our drawing tools).

See the examples section of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan2 for the edge case values around the `-180` `180` branch cut.